.changeset/afraid-fans-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
```

Comment:
I originally had it under `minor` but [moved it](https://github.com/iTwin/iTwinUI/pull/2185/commits/aa574ab7c6a19539316663bc3df0ce4cc2628b5a) to clean up the changelog because I think this is an implementation detail. It will be released in a minor either way.